src/components/subscriptions/licenses/LicenseAllocationHeader.jsx

```diff
@@ -24,26 +17,18 @@ const LicenseAllocationHeader = () => {
         {' of '}
         {subscription.licenses?.total} licenses allocated
       </p>
-      <div className="my-3 row">
-        <div className="col-12 col-md-5 mb-3 mb-md-0">
+      <Row className="justify-content-between">
+        <Col lg={6} xs={12} className="mt-2">
```

Comment:
nit: if both columns require a top margin, might consider putting `mt-2` on the parent `Row` instead.

Reply:
this top margin was included to handle the scenario where the columns stack (e.g. mobile view) - it adds margin in between the search bar and download button. As I'm writing this I now realize I could use a bottom margin on the search bar and that should accomplish the same thing 👍